src/IO/ReadHelpers.h

```diff
@@ -1578,4 +1578,7 @@ void readQuotedField(String & s, ReadBuffer & buf);
 void readJSONField(String & s, ReadBuffer & buf);
 
 void readTSVField(String & s, ReadBuffer & buf);
+
+bool parseComplexEscapeSequence(String & s, ReadBuffer & buf);
```

Comment:
Please add a comment, what exactly is cosidered as `EscapeSequence` what is `Complex` one.

Reply:
Ok. This method already exists, it just does not have a public API. I'll copy its implementation comment if it's alright:

```
/** Parse the escape sequence, which can be simple (one character after backslash) or more complex (multiple characters).
  * It is assumed that the cursor is located on the `\` symbol
  */
```